src/main/java/graphql/annotations/dataFetchers/GraphQLUndefined.java

```diff
@@ -15,12 +15,35 @@
 import graphql.schema.GraphQLSchemaElement;
 import graphql.schema.GraphQLType;
 import graphql.schema.GraphQLTypeVisitor;
+import graphql.schema.SchemaElementChildrenContainer;
 import graphql.util.TraversalControl;
 import graphql.util.TraverserContext;
 
+import java.util.List;
+
 public class GraphQLUndefined implements GraphQLType {
+    @Override
+    public List<GraphQLSchemaElement> getChildren() {
+        return GraphQLType.super.getChildren();
+    }
+
+    @Override
+    public SchemaElementChildrenContainer getChildrenWithTypeReferences() {
+        return GraphQLType.super.getChildrenWithTypeReferences();
+    }
+
+    @Override
+    public GraphQLSchemaElement withNewChildren(SchemaElementChildrenContainer newChildren) {
+        return GraphQLType.super.withNewChildren(newChildren);
+    }
+
     @Override
     public TraversalControl accept(TraverserContext<GraphQLSchemaElement> context, GraphQLTypeVisitor visitor) {
         return null;
     }
+
+    @Override
+    public GraphQLSchemaElement copy() {
+        return null;
```

Comment:
I don't think this should return null https://github.com/graphql-java/graphql-java/blob/066c751e01120a9b29531a698769e72fb99bcb67/src/main/java/graphql/schema/GraphQLSchemaElement.java#L59

Reply:
Fixed